### PR TITLE
Update logging to reference fluent-bit

### PR DESCRIPTION
**Pull request by philipnrmn**
## Description
[DCOS-51036](https://jira.mesosphere.com/browse/DCOS-51036) - Documentation fluent-bit on docs.mesosphere.com

## Outstanding Work
 - [x] Modify the Overview section, adding a section which explains:
   - How logs can enter the pipeline
   - How fluent bit functions as the log pipeline in DC/OS
 - [x] ~Quickstart~
   - ~Add a “Hello World” Example (outputting logs to ES/Kibana). Avoid replicating the tutorial content here.~
   - Skipping, not doable without replicating tutorial content.
 - [x] ~Tutorials~
   - ~Modify the 'log management with elk' and 'log management with splunk' sections to use fluent bit instead of journalbeat / splunk-forwarder~
   - Skipping due to https://jira.mesosphere.com/browse/DCOS_OSS-5109
 - [x] Reference docs
   - Rename them from Logging API to Logging Reference
   - Document the fluentbit task labels to get Mesos tasks sending logs to Fluentbit
   - Document the tags which fluent bit adds

## Urgency
- [x] High

## Requirements
- Test all commands and procedures.
- Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects).
- Change all affected versions (e.g. 1.7, 1.8, 1.9, 1.10, 1.11, 1.12).
- See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing).

### CODE REVIEW

File: pages/1.13/monitoring/logging/aggregating/elk/index.md [hunk 1]
```diff
@@ -10,144 +10,82 @@ enterprise: false
 
 
 
-You can pipe system and application logs from the nodes in a DC/OS cluster to an Elasticsearch server. These instructions are based on CentOS 7 and might differ substantially from other Linux distributions.
+You can pipe system and application logs from the nodes in a DC/OS cluster to an Elasticsearch server.
 
 
 # What this document does and does not cover
```
Comment (anithamuthu79) [inline pages/1.13/monitoring/logging/aggregating/elk/index.md]: Rename the section title to `Streaming Filebeat into Elasticsearch`

File: pages/1.13/monitoring/logging/aggregating/elk/index.md [hunk 2]
```diff
@@ -10,144 +10,82 @@ enterprise: false
...
 
-This document describes how to send Filebeat output from each node to a centralized Elasticsearch instance. This document describes how to directly stream from Filebeat into Elasticsearch. Logstash is not used in this architecture. If you are interested in filtering, parsing and understanding the logs with an intermediate Logstash stage, see the Logstash [documentation][8] and the example in [Filtering logs with ELK][3].
+This document describes how to send Fluent Bit output from each node to a centralized Elasticsearch instance. This document describes how to directly stream from Fluent Bit into Elasticsearch. Logstash is not used in this architecture. If you are interested in filtering, parsing and understanding the logs with an intermediate Logstash stage, see the Logstash [documentation][4] and the example in [Filtering logs with ELK][2].
```
Comment (anithamuthu79) [inline pages/1.13/monitoring/logging/aggregating/elk/index.md]: Rewrite `This document describes how to send Fluent Bit output from each node to a centralized Elasticsearch instance. This document describes how to directly stream from Fluent Bit into Elasticsearch. Logstash is not used in this architecture. If you are interested in filtering, parsing and understanding the logs with an intermediate Logstash stage, see the Logstash [documentation][4] and the example in [Filtering logs with ELK][2].`

with `This page describes how to send Fluent Bit output from each node to a centralized Elasticsearch instance. It also describes how to directly stream from Fluent Bit into Elasticsearch. Logstash is not used in this architecture. If you are interested in filtering, parsing, and understanding the logs with an intermediate Logstash stage, see the Logstash [documentation][4] and the example in [Filtering logs with ELK][2].

File: pages/1.13/monitoring/logging/aggregating/elk/index.md [hunk 3]
```diff
@@ -10,144 +10,82 @@ enterprise: false
...
 
-This document does not explain how to set up and configure an Elasticsearch server. This document does not describe how to set up secure TLS communication between the Filebeat instances and Elasticsearch. For details on how to achieve this, see the [Filebeat][2] and [Elasticsearch][5] documentation.
+This document does not explain how to set up and configure an Elasticsearch server. This document does not describe how to set up secure TLS communication between the Fluent Bit instances and Elasticsearch. For details on how to achieve this, see the [Fluent Bit][1] and [Elasticsearch][3] documentation.
```
Comment (anithamuthu79) [inline pages/1.13/monitoring/logging/aggregating/elk/index.md]: <p class="message--warning"><strong>WARNING: </strong>Instructions on how to set up and configure an Elasticsearch server, secure TLS communication between the Fluent Bit instances and Elasticsearch are not explained here. You can see the [Fluent Bit][1] and [Elasticsearch][3] documentation for more information to achieve this task. </p>

File: pages/1.13/monitoring/logging/aggregating/elk/index.md [hunk 4]
```diff
@@ -10,144 +10,82 @@ enterprise: false
...
 
 **Prerequisites**
 
 *   An existing Elasticsearch installation that can ingest data for indexing
-*   All DC/OS nodes must be able to connect to your Elasticsearch server on the port used for communication between Elasticsearch and Filebeat (9200 by default)
+*   All DC/OS nodes must be able to connect to your Elasticsearch server on the port used for communication between Elasticsearch and Fluent Bit (9200 by default)
+*   A location on each DC/OS node for your custom Fluent Bit config. This tutorial will use `/etc/fluent-bit/`.
 
-## <a name="all"></a>Step 1: Install Filebeat
+## Step 1: Master nodes
```
Comment (anithamuthu79) [inline pages/1.13/monitoring/logging/aggregating/elk/index.md]: Remove the text `Step 1:`

File: pages/1.13/monitoring/logging/aggregating/elk/index.md [hunk 5]
```diff
@@ -10,144 +10,82 @@ enterprise: false
...
+@INCLUDE /opt/mesosphere/etc/fluent-bit/master.conf
+[OUTPUT]
+     Name es
+     Match *
+     Host <Elasticsearch host>
+     Port <Elasticsearch port>
+```
 
-1.  Populate a new `filebeat.yml` configuration file, including an additional input entry for the file `/var/log/dcos/dcos.log`. The additional log file will be used to capture the DC/OS logs in a later step. Remember to substitute the variables `$ELK_HOSTNAME` and `$ELK_PORT` below for the actual values of the host and port where your Elasticsearch is listening on.
+## Step 2: Agent nodes
```
Comment (anithamuthu79) [inline pages/1.13/monitoring/logging/aggregating/elk/index.md]: Remove the text `Step 2:`

File: pages/1.13/monitoring/logging/aggregating/elk/index.md [hunk 6]
```diff
@@ -10,144 +10,82 @@ enterprise: false
...
+@INCLUDE /opt/mesosphere/etc/fluent-bit/agent.conf
+[OUTPUT]
+     Name es
+     Match *
+     Host <Elasticsearch host>
+     Port <Elasticsearch port>
+```
 
-## <a name="all-2"></a>Step 2: Set up service for parsing the journal
+## Step 3: All nodes
```
Comment (anithamuthu79) [inline pages/1.13/monitoring/logging/aggregating/elk/index.md]: Remove the text `Step 3:`

File: pages/1.13/monitoring/logging/aggregating/splunk/index.md [hunk 7]
```diff
@@ -9,129 +9,73 @@ enterprise: false
 ---
 
 # Overview
-You can pipe system and application logs from a DC/OS cluster to your existing Splunk server. This document describes how to configure a Splunk universal forwarder to send output from each node to a Splunk installation. This document does not explain how to set up and configure a Splunk server.
+You can pipe system and application logs from a DC/OS cluster to your existing Splunk server. This document describes how to configure Fluent Bit to send output from each node to a Splunk installation. This document does not explain how to set up and configure a Splunk server.
```
Comment (anithamuthu79) [inline pages/1.13/monitoring/logging/aggregating/splunk/index.md]: Replace `This document describes how to configure Fluent Bit to send output from each node to a Splunk installation. This document does not explain how to set up and configure a Splunk server.`

with `This page describes how to configure Fluent Bit to send output from each node to a Splunk installation. It does not explain how to set up and configure a Splunk server.

File: pages/1.13/monitoring/logging/aggregating/elk/index.md [hunk 8]
```diff
@@ -10,144 +10,82 @@ enterprise: false
...
-  copytruncate
-  rotate 5
-  compress
-  compresscmd /bin/xz
-}
+$ sudo systemctl daemon-reload
+$ sudo systemctl restart dcos-fluent-bit.service
 ```
 
 # What's Next
```
Comment (anithamuthu79) [inline pages/1.13/monitoring/logging/aggregating/elk/index.md]: Remove this line.

File: pages/1.13/monitoring/logging/aggregating/elk/index.md [hunk 9]
```diff
@@ -10,144 +10,82 @@ enterprise: false
...
 
-For details on how to filter your logs with ELK, see [Filtering DC/OS logs with ELK][3].
+For details on how to filter your logs with ELK, see [Filtering DC/OS logs with ELK][2].
```
Comment (anithamuthu79) [inline pages/1.13/monitoring/logging/aggregating/elk/index.md]: Replace `For details` with `For more information`

File: pages/1.13/monitoring/logging/aggregating/splunk/index.md [hunk 10]
```diff
@@ -9,129 +9,73 @@ enterprise: false
...
-
 **Prerequisites**
 
 *   An existing Splunk installation that can ingest data for indexing
-*   All DC/OS nodes must be able to connect to your Splunk indexer via HTTP or HTTPS
-*   The `ulimit` of open files must be set to `unlimited` for your user with root access.
+*   All DC/OS nodes must be able to connect to your Splunk `indexer via HTTP or HTTPS
+*   A location on each DC/OS node for your custom Fluent Bit config. This tutorial will use `/etc/fluent-bit/`.
+
+## Step 1: Master nodes
```
Comment (anithamuthu79) [inline pages/1.13/monitoring/logging/aggregating/splunk/index.md]: Remove `Step 1`.

File: pages/1.13/monitoring/logging/aggregating/splunk/index.md [hunk 11]
```diff
@@ -9,129 +9,73 @@ enterprise: false
...
+@INCLUDE /opt/mesosphere/etc/fluent-bit/master.conf
+[OUTPUT]
+    Name splunk
+    Match *
+    Host <Splunk server host>
+    Port <Splunk server port>
+    Splunk_Token <Splunk HTTP event collector token>
+```
+
+## Step 2: Agent nodes
```
Comment (anithamuthu79) [inline pages/1.13/monitoring/logging/aggregating/splunk/index.md]: Remove `Step 2`.

File: pages/1.13/monitoring/logging/aggregating/splunk/index.md [hunk 12]
```diff
@@ -9,129 +9,73 @@ enterprise: false
...
+@INCLUDE /opt/mesosphere/etc/fluent-bit/agent.conf
+[OUTPUT]
+    Name splunk
+    Match *
+    Host <Splunk server host>
+    Port <Splunk server port>
+    Splunk_Token <Splunk HTTP event collector token>
+```
+
+## Step 3: All nodes
```
Comment (anithamuthu79) [inline pages/1.13/monitoring/logging/aggregating/splunk/index.md]: Remove `Step 3`.

File: pages/1.13/monitoring/logging/aggregating/splunk/index.md [hunk 13]
```diff
@@ -9,129 +9,73 @@ enterprise: false
...
+```
 
+4.  Reload systemd to update `dcos-fluent-bit.service`, and restart it:
 
+```
+$ sudo systemctl daemon-reload
+$ sudo systemctl restart dcos-fluent-bit.service
+```
 
 # What's next
```
Comment (anithamuthu79) [inline pages/1.13/monitoring/logging/aggregating/splunk/index.md]: Remove this line(#80).

File: pages/1.13/monitoring/logging/logging-api-examples/index.md [hunk 14]
```diff
@@ -59,7 +59,7 @@ curl -k -H "Authorization: token=${DCOS_AUTH_TOKEN}" "${DCOS_URL}/system/v1/agen
 
 # Container Logs
 
-**Important:** The following example requires journald task logging, which by default is [disabled](/1.13/monitoring/logging/logging-api/#compatibility).
+**Important:** The following example requires journald task logging, which by default is [disabled](/1.13/monitoring/logging/logging-reference/#compatibility).
```
Comment (anithamuthu79) [inline pages/1.13/monitoring/logging/logging-api-examples/index.md]: Use the format for `Important message` from https://wiki.mesosphere.com/display/DOCS/Documentation+Style+Guide#DocumentationStyleGuide-Notes

**Comment by branden** [comment]
@anithamuthu79 Thanks for the review, I pushed an update. I removed the ELK and Splunk tutorial updates due to https://jira.mesosphere.com/browse/DCOS_OSS-5109, so that resolves most of your comments.

File: pages/1.13/monitoring/logging/index.md [hunk 15]
```diff
@@ -64,7 +64,11 @@ Figure 1. System health log showing nodes
 
 # Log aggregation
 
-Streaming logs from machines in your cluster is not always viable. Sometimes you need a history of what has happened, which requires a method for storing and aggregating logs. These topics describe some of the most common solutions:
+Streaming logs from machines in your cluster is not always the best solution for examing events and debugging issues.
```
Comment (gracedo) [inline pages/1.13/monitoring/logging/index.md]: typo `examining`

**Pull request merged**